Common/Core/TrackSelection.h

```diff
@@ -52,26 +52,83 @@ class TrackSelection
   template <typename T>
   bool IsSelected(T const& track)
   {
-    const bool isRun2 = track.trackType() == o2::aod::track::Run2Track || track.trackType() == o2::aod::track::Run2Tracklet;
```

Comment:
@njacazio maybe this line shall be left as it is, so that the class is still usable with Run2 converted data

Reply:
This should happen already here, before it was split in two occurrences, now it has been unified, have a look at line 138